Source/controls/plrctrls.cpp

```diff
@@ -1991,6 +1991,7 @@ void PerformSecondaryAction()
 
 void QuickCast(size_t slot)
 {
+	MouseActionType PrevMouseButtonAction = LastMouseButtonAction;

```

Comment:
We use uppercase for globals and lowercase for locals :)

```suggestion
	MouseActionType prevMouseButtonAction = LastMouseButtonAction;
```